src/Controller/SubscriptionController.php

```diff
@@ -136,7 +136,9 @@ public function subscribe($entity_type_id, EntityInterface $group, OgMembershipT
       return new RedirectResponse($group->toUrl()->setAbsolute(TRUE)->toString());
     }
 
-    if (!$this->ogAccess->userAccess($group, 'subscribe', $user) && !$this->ogAccess->userAccess($group, 'subscribe without approval', $user)) {
+    $subscribe = $this->ogAccess->userAccess($group, 'subscribe', $user);
+    $subscribeWithoutApproval = $this->ogAccess->userAccess($group, 'subscribe without approval', $user);
```

Comment:
We don't really need to pass the user if it's the current user either:
```suggestion
    $subscribe = $this->ogAccess->userAccess($group, 'subscribe');
    $subscribeWithoutApproval = $this->ogAccess->userAccess($group, 'subscribe without approval');
```